Source/SwiftLintFramework/LintOrAnalyzeCommand.swift

```diff
@@ -145,12 +145,16 @@ package struct LintOrAnalyzeCommand {
         if let baselineOutputPath = options.writeBaseline ?? builder.configuration.writeBaseline {
             try Baseline(violations: builder.unfilteredViolations).write(toPath: baselineOutputPath)
         }
+        var numberOfSeriousViolations = 0
```

Comment:
This will become an issue when we enforce strict concurrency checks. The API should allow to write this as:
```swift
let numberOfSeriousViolations = Signposts.record(...) {
    try postProcessViolations(files: files, builder: builder)
}
```